### PR TITLE
update to DSE 6.8.34

### LINKED ISSUE #2534: Update to DSE 6.8.34
A new version of DSE has been released and should be used.

### PR DESCRIPTION
**Pull request by jeffreyscarpenter**
Fixes #2534 

**Checklist**
- [x] Changes manually tested
- [ ] Automated Tests added/updated
- [ ] Documentation added/updated
- [x] CLA Signed: [DataStax CLA](https://cla.datastax.com/)

### CODE REVIEW

**Comment by tatu-at-datastax** [comment]
Related to issue #1889 (DSE backend updated `geometry-api`?)

File: coordinator/persistence-dse-6.8/pom.xml [hunk 1]
```diff
@@ -135,7 +135,7 @@
     <dependency>
       <groupId>io.netty</groupId>
       <artifactId>netty-codec</artifactId>
-      <version>4.1.78.1.dse</version>
+      <version>4.1.86.1.dse</version>
```
Comment (tatu-at-datastax) [inline coordinator/persistence-dse-6.8/pom.xml:138]: Updated as per DSE 6.8.34 release notes (may also resolve Netty CVEs, not 100% sure)

**Comment by tatu-at-datastax** [comment]
Ok, C*3.11 test failed first but passed on re-try.

**Review by tatu-at-datastax — APPROVED** [review]

**Comment by ivansenic** [comment]
@jeffreyscarpenter @tatu-at-datastax So this is only done on v2? No updates for the v1?

**Comment by tatu-at-datastax** [comment]
Correct. We are now trying to focus on v2 as far as I understand. Even for DSE updates.

**Pull request merged**